src/test/features/claimant-response/helpers/taskListBuilder.ts

```diff
@@ -742,11 +742,8 @@ describe('Claimant response task list builder', () => {
         draft, claim, new MediationDraft()
       )
 
-      if (FeatureToggles.isEnabled('mediation')) {
-        expect(taskList.tasks.find(task => task.name === 'Free telephone mediation')).not.to.be.undefined
-      } else {
-        expect(taskList).to.be.eq(undefined)
-      }
+      expect(taskList.tasks.find(task => task.name === 'Free telephone mediation')).not.to.be.undefined
+
```

Comment:
I see that you removed mediation feature toggle, do we not need it? does this test work with and without mediation toggle?